src/app/submission/sections/sections.service.ts

```diff
@@ -173,8 +173,31 @@ export class SectionsService {
     );
   }
 
+  getShownSectionErrors(submissionId: string, sectionId: string, sectionType: SectionsType): Observable<SubmissionSectionError[]> {
```

Comment:
Can we add TypeDocs to this method please?

Reply:
done